validationTemplate/src/ModelCalibrationTemplate/calibration_incomplete_XML.xml

```diff
@@ -0,0 +1,96 @@
+<Simulation verbosity="debug">
```

Comment:
(note about files above this one) it looks like some `__pycache__` binaries are included; you'll want to remove those from the PR.